public/components/wz-menu/wz-menu.js

```diff
@@ -171,16 +173,22 @@ class WzMenu extends Component {
 
 
   async componentDidUpdate(prevProps) {
+
     if (this.state.APIlist && !this.state.APIlist.length) {
       this.loadApiList();
     }
     const { id: apiId } = JSON.parse(AppState.getCurrentAPI());
     const { currentAPI } = this.state;
     const currentTab = this.getCurrentTab();
+    
     if (currentTab !== this.state.currentMenuTab) {
       this.setState({ currentMenuTab: currentTab });
     }
 
+    if(this.props.windowSize){
+      this.showSelectorsInPopover = this.props.windowSize.width < 1100 ? true : false;
```

Comment:
This could be just like:
`this.showSelectorsInPopover = this.props.windowSize.width < 1100`